components/signup/signup_controller/signup_controller.test.jsx

```diff
@@ -1,126 +0,0 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
-// See LICENSE.txt for license information.
-
-import {shallow} from 'enzyme';
-import React from 'react';
-
-import * as GlobalActions from 'actions/global_actions';
-import {browserHistory} from 'utils/browser_history';
-import {Constants} from 'utils/constants';
-
-import SignupController from './signup_controller.jsx';
-
-jest.mock('actions/global_actions', () => ({
-    redirectUserToDefaultTeam: jest.fn(),
-}));
-
-jest.mock('utils/browser_history', () => ({
-    browserHistory: {
-        push: jest.fn(),
-    },
-}));
-
-describe('components/SignupController', () => {
-    const baseProps = {
-        location: {
-            search: '',
-            pathname: '/signup_user_complete/',
-            hash: '',
-        },
-        noAccounts: false,
-        loggedIn: true,
-        isLicensed: true,
-        isCloud: false,
-        enableOpenServer: true,
-        enableSAML: true,
-        enableLDAP: true,
-        enableSignUpWithEmail: true,
-        enableSignUpWithGitLab: true,
-        enableSignUpWithGoogle: true,
-        enableSignUpWithOffice365: true,
-        enableSignUpWithOpenId: true,
-        openidButtonText: 'OpenId',
-        openidButtonColor: '#FFFFFF',
-        samlLoginButtonText: 'SAML',
-        ldapLoginFieldName: '',
-        actions: {
-            removeGlobalItem: jest.fn(),
-            getTeamInviteInfo: jest.fn(),
-            addUserToTeamFromInvite: jest.fn(),
-        },
-    };
-
-    test('should match snapshot for all signup options enabled with isLicensed enabled', () => {
```

Comment:
Did we migrate these unit tests?

Reply:
I'm still working on unit and e2e testing, those will be added.